tests/figures/chart/menu_item_insert_chart.test.ts

```diff
@@ -128,19 +129,28 @@ describe("Insert chart menu item", () => {
     const { width, height } = model.getters.getSheetViewDimension();
     const payload = { ...defaultPayload };
     payload.definition = expect.any(Object);
-    payload.position = {
-      x: (width - DEFAULT_FIGURE_WIDTH) / 2,
-      y: (height - DEFAULT_FIGURE_HEIGHT) / 2,
+    payload.offset = {
+      x: 40,
+      y: 10.5,
+    };
+    payload.anchor = {
+      col: 2,
+      row: 14,
```

Comment:
All the changes to this test file are sus'. What are these hardcoded values and where do they come from ? I get it's supposed to be the center of the screen, but why do we need to hard code them instead of computing it now ? And why is another test `expect({ x: figureUI.x, y: figureUI.y }).toStrictEqual` now needed ? Is the payload test not enough ? I'm a bit confused.